core/examples/luigi-sample-angular/src/assets/extendedConfiguration.js

```diff
@@ -419,7 +425,8 @@ Luigi.setConfig({
         {
           pathSegment: 'overview',
           label: 'Overview',
-          viewUrl: '/sampleapp.html#/overview'
+          viewUrl: '/sampleapp.html#/overview',
+          navCollapse: true
```

Comment:
It would be fine for me to leave it, but then please add some left margin to the headline in the content area
![in the content area](https://user-images.githubusercontent.com/8238188/49240013-7ed90b00-f404-11e8-8877-028604e5db75.png)